components/combobox/combobox.jsx

```diff
@@ -253,6 +259,10 @@ const propTypes = {
 	 * This callback exposes the selected listbox reference / DOM node to parent components.
 	 */
 	selectedListboxRef: PropTypes.func,
+	/**
+	 * Accepts a tooltip that is displayed when hovering on disabled menu items.
+	 */
+	tooltipMenuItemDisabled: PropTypes.element,
```

Comment:
I need testing that this works. A story/snapshot is fine.

Reply:
Yup - added a Mocha test in `/combobox/__test__/combobox.browser-test.jsx`